### DISCUSSION TITLE
Requires Flutter 3

### DISCUSSION THREAD

**Discussion by steveneal** [gdelataillade/alarm#7] [General]
Hi,

Firstly, thanks for implementing this package.

You have a dependency on flutter_local_notifications ^13.0.0, which in turn has a minimum Flutter version requrement of v3. See their pubspec file:

environment:
  sdk: ">=2.15.0 <3.0.0"
  flutter: ">=3.0.0"

This means that we cannot use your lib in our project as other libs we depend don't (and are not likely in the near future) to support Flutter v3. 

Is there a reason for this or is it possible to ensure that your lib is compatible with Flutter v2?

Thanks again.

Steve.

**Comment by gdelataillade** [comment]
Hi @steveneal ,

Thank you for your interest in the package.

The main reason we chose to use `flutter_local_notifications ^13.0.0` is because we want to stay up to date with all the dependencies. This way, we have access to new features, bug fixes and we make sure that we don't have dependencies version conflicts.

However, if you need to use this package with Flutter v2, you can fork the repository and downgrade flutter_local_notifications to a compatible version. If you chose to do this, please come back sometimes to check `alarm` package new updates.

Let me know if you have any other questions.

Best regards,

Gautier
